groups/wg-etcd-operator/groups.yaml

```diff
@@ -0,0 +1,49 @@
+groups:
+  #
+  # Mailing lists
+  #
+  # Each group here represents a mailing list for the WG or its subprojects,
+  # and is not intended to govern access to infrastructure
+  #
+
+  #
+  # WG etcd-operator mailing lists
+  #
+
+  - email-id: wg-etcd-operator-leads@kubernetes.io
+    name: wg-etcd-operator-leads
+    description: |-
+      WG etcd-operator Organizers
+    settings:
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      AllowWebPosting: "true"
+      ReconcileMembers: "true"
+      WhoCanModerateContent: "OWNERS_AND_MANAGERS"
+    owners:
+      - benjamin.ahrtr@gmail.com
+      - ciprianhacman@gmail.com
+      - jberkus@redhat.com
+      - jablair@redhat.com
+      - justinsb@google.com
+
+  - email-id: wg-etcd-operator@kubernetes.io
+    name: wg-etcd-operator
+    description: |-
+      Public mailing list for WG etcd-operator
+    settings:
+      WhoCanJoin: "ANYONE_CAN_JOIN"
+      WhoCanViewGroup: "ANYONE_CAN_VIEW"
+      WhoCanDiscoverGroup: "ANYONE_CAN_DISCOVER"
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      AllowWebPosting: "true"
+      MessageModerationLevel: "MODERATE_NON_MEMBERS"
```

Comment:
Does this mean new messages will need to be approved? I.e. the same as `etcd-dev` mailing list?  We've had a lot of issues with spam which has meant we need to approve new messages rather than letting anyone send messages directly.

Reply:
> All messages from non group members are sent to the group owner's email address for approval. If approved, the message is sent to the group.

Reply:
For etcd-dev, we moderate all messages, including members and non-members. For wg-etcd-operator, `MODERATE_NON_MEMBERS` only moderate messages from non members. Should we moderate all messages as well? @hakman @jberkus @jmhbnz @justinsb @neolit123

If there is no comment or objection, then let's just keep it as it's.